frontend/src/views/discover/discover.css

```
@@ -0,0 +1,1089 @@
+  font-size: 11.5px;
+  font-weight: 600;
+  text-transform: capitalize;
+}
+.cp-content-hero-title {
+  margin: 0;
+  color: var(--text);
+  font-size: 30px;
+  font-weight: 700;
+  line-height: 1.08;
+  letter-spacing: 0;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)